common/app/model/dotcomrendering/pageElements/PageElement.scala

```diff
@@ -335,11 +338,9 @@ object PageElement {
           }
 
           case Some(interactive: InteractiveAtom) => {
-            Some(InteractiveMarkupBlockElement(
-              id = interactive.id,
-              html = Some(interactive.html),
-              css = Some(interactive.css),
-              js = interactive.mainJS
+            val encodedId = URLEncoder.encode(interactive.id, "UTF-8")
+            Some(InteractiveUrlBlockElement(
+              url = s"${Configuration.ajax.url}/embed/atom/interactive/$encodedId"
```

Comment:
Is the 's' prefix right?

Reply:
Good question! Yeah - it enables string interpolation: https://docs.scala-lang.org/overviews/core/string-interpolation.html

Reply:
yeah it's a weird Scala thing.